mne/preprocessing/eyetracking/calibration.py

```diff
@@ -0,0 +1,289 @@
+    screen_size : array-like
+        The width and height (in meters) of the screen that the eyetracking data was
+        collected  with. For example ``(.531, .298)`` for a monitor with a display area
+        of 531 x 298 mm.
+    screen_distance : float
+        The distance (in meters) from the participant's eyes to the screen.
+    screen_resolution : array-like
+        The resolution (in pixels) of the screen that the eyetracking data was
+        collected with. For example, ``(1920, 1080)`` for a 1920x1080 resolution
+        display.
+    """
+
+    def __init__(
+        self,
+        onset=None,
+        model=None,
+        eye=None,
+        avg_error=None,
+        max_error=None,
+        positions=None,
+        offsets=None,
+        gaze=None,
+        screen_size=None,
+        screen_distance=None,
+        screen_resolution=None,
+    ):
+        super().__init__(
+            onset=onset,
+            model=model,
+            eye=eye,
+            avg_error=avg_error,
+            max_error=max_error,
+            screen_size=screen_size,
+            screen_distance=screen_distance,
+            screen_resolution=screen_resolution,
+        )
+        self["positions"] = positions
+        self["offsets"] = offsets
+        self["gaze"] = gaze
+
+    def __repr__(self):
+        """Return a summary of the Calibration object."""
+        onset = self.get("onset", "N/A")
+        model = self.get("model", "N/A")
+        eye = self.get("eye", "N/A")
+        avg_error = self.get("avg_error", "N/A")
+        max_error = self.get("max_error", "N/A")
+        screen_size = self.get("screen_size", "N/A")
+        screen_distance = self.get("screen_distance", "N/A")
+        screen_resolution = self.get("screen_resolution", "N/A")
+        return (
+            f"Calibration |\n"
+            f"  onset: {onset} seconds\n"
+            f"  model: {model}\n"
+            f"  eye: {eye}\n"
+            f"  average error: {avg_error} degrees\n"
+            f"  max error: {max_error} degrees\n"
+            f"  screen size: {screen_size} meters\n"
+            f"  screen distance: {screen_distance} meters\n"
+            f"  screen resolution: {screen_resolution} pixels\n"
+        )
+
+    def copy(self):
+        """Copy the instance.
+
+        Returns
+        -------
+        cal : instance of Calibration
+            The copied Calibration.
+        """
+        return deepcopy(self)
+
+    def __setitem__(self, key, value):
+        """Make sure that some keys are caste as numpy arrays.
+
+        Because methods like plot expect numpy arrays.
+        """
+        if key in ("positions", "offsets", "gaze") and isinstance(value, (tuple, list)):
+            logger.info("Converting %s to numpy array", key)
+            value = np.array(value)
+        super().__setitem__(key, value)
+
+    def plot(self, title=None, show_offsets=False, origin="top-left", show=True):
```

Comment:
In the tutorial you do this and in general it's more informative so my vote would be for `show_offsets=True` by default
```suggestion
    def plot(self, title=None, show_offsets=True, origin="top-left", show=True):
```

Reply:
good idea!